generative_data_prep/data_prep/pipeline.py

```diff
@@ -654,6 +674,16 @@ def pipeline_main(  # noqa: C901
     if not keep_split_jsonls:
         shutil.rmtree(split_dir)
 
+    file_names = []
+    for file_name in os.listdir(json_error_log_dir):
+        file_names.append(os.path.join(json_error_log_dir, file_name))
+    with open(os.path.join(output_dir, "json_error_log.log"), "w") as outfile:
```

Comment:
nit: Maybe change the name to be more user friendly? 

Like json_load_failed_lines.log ? Not sure if this is better haha